### PR TITLE
first pass at tutorial doc

### PR DESCRIPTION
**Pull request by christian-posta**

### CODE REVIEW

File: docs/content/tutorial_code/getting_started_1.md [hunk 1]
```diff
@@ -1,7 +1,321 @@
...
+### Deploying Envoy
+
+In this tutorial, we'll use Gloo, an API Gateway based on Envoy that has built-in wasm support but these steps should also work for base Envoy. 
+
+```shell
+$  kubectl apply -f gloo.yaml
+```
+
+*****
+This section needs love... it's not working yet ^^ we need a nice way to install Gloo
```
Comment (EItanya) [inline docs/content/tutorial_code/getting_started_1.md]: Can use helm for sure. The version of gloo will be `1.2.5`

File: docs/content/tutorial_code/getting_started_1.md [hunk 2]
```diff
@@ -1,7 +1,321 @@
...
+
+```shell
+$  kubectl apply -f default-virtualservice.yaml
+```
+
+To get Gloo's external IP, run the following:
+
+```shell
+$  URL=$(kubectl get svc -n gloo-system gateway-proxy \
+ -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
```
Comment (EItanya) [inline docs/content/tutorial_code/getting_started_1.md]: wanna use glooctl here?

File: docs/content/tutorial_code/getting_started_1.md [hunk 3]
```diff
@@ -1,7 +1,321 @@
...
+    plugins:
+      extensions:
+        configs:
+          wasm:
+            image: webassemblyhub.io/christian-posta/test:v0.1
+            name: christian
+            root_id: add_header_root_id
+  proxyNames:
+  - gateway-proxy-v2
+  useProxyProto: false
```
Comment (EItanya) [inline docs/content/tutorial_code/getting_started_1.md]: the master README.md has an updated version of what this should look like for the new version of gloo

File: docs/content/tutorial_code/getting_started_1.md (hunk 1 above)
Comment (christian-posta) [reply docs/content/tutorial_code/getting_started_1.md]: No, we don't want to use helm for this guide. We want to use as simple of a kube resource as possible. I'm trying to stay away from glooctl as well

File: docs/content/tutorial_code/getting_started_1.md (hunk 2 above)
Comment (christian-posta) [reply docs/content/tutorial_code/getting_started_1.md]: No. We want this as simple as possible with no additional dependencies.

File: docs/content/tutorial_code/getting_started_1.md (hunk 3 above)
Comment (christian-posta) [reply docs/content/tutorial_code/getting_started_1.md]: Okay, yah, I need to update for that.

**Pull request merged**